grails-app/migrations/0.9.x/changelog-2025-02-06-0000-add-column-status-to-cycle-count.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog/1.9"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9 http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
+
+    <changeSet author="ewaterman" id="060220250000-0">
+        <preConditions onFail="MARK_RAN">
+            <not>
+                <columnExists tableName="cycle_count" columnName="status"/>
+            </not>
+        </preConditions>
+
+        <addColumn tableName="cycle_count">
+            <column name="status" type="VARCHAR(100)">
+                <constraints nullable="false" />
```

Comment:
TODO: set a default value for existing data! Or just truncate obdev3 and 5 because this isn't live yet??

Reply:
We've run once recently into a similar issue. We've just cleared the data (via the script I've sent on Slack for you some time ago). I'm fine with doing the same again.